### PR TITLE
updated medicaid share to total computable medicaid cost

### LINKED ISSUE #2324: Update areas in the app where "total computable" should be used
Total Computable cost is a term used in the new tables and should be consistently applied throughout the app.

### This task is done when...
 - [ ] The app has been reviewed and areas where "total computable" should be used have been identified
 - [ ] The team is in agreement on changes to identified areas

### PR DESCRIPTION
**Pull request by thetif**
Updated areas in the app where medicaid share was used
resolves #2324

### This pull request changes...

- resolves the instances suggested in #2324 

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
~- [ ] The change has been documented~
~- [ ] Associated OpenAPI documentation has been updated~
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by thetif** [comment]
@CSwartzHMA @beparticular The app has changed a lot since this ticket was made, can you make sure I updated everything correctly, please?

**Review by CSwartzHMA — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,43 +4,14 @@ Anticipated release: October 26th, 2020
...
-[#2141]: https://github.com/CMSgov/eAPD/issues/2141
-[#2142]: https://github.com/CMSgov/eAPD/issues/2142
-[#2306]: https://github.com/CMSgov/eAPD/issues/2306
-[#2429]: https://github.com/CMSgov/eAPD/issues/2429
-[#2545]: https://github.com/CMSgov/eAPD/issues/2545
-[#2501]: https://github.com/CMSgov/eAPD/issues/2501
-[#2502]: https://github.com/CMSgov/eAPD/issues/2502
-[#2503]: https://github.com/CMSgov/eAPD/issues/2503
-[#2509]: https://github.com/CMSgov/eAPD/issues/2509
-[#2521]: https://github.com/CMSgov/eAPD/issues/2521
```
Comment (radavis) [inline CHANGELOG.md:46]: Looks like there's some text in `/Users/rd/fearless/eapd/web/src/i18n/locales/en/apd/stateProfile.yaml` that still refers to "State Cost Categories"

**Review by radavis — APPROVED** [review]
My only other thought here is that we may want to update the URL from `state-costs` to `state-staff`. Your call.

**Review by tbolt — APPROVED** [review]
Looks great

File: CHANGELOG.md (hunk 1 above)
Comment (thetif) [reply CHANGELOG.md:46]: oh thanks, I'll check it out and I'll change the route too

Comment (thetif) [reply CHANGELOG.md:46]: actually I'm going to leave the route, because the next one is contractor-costs, I think leaving it lines up better

**Review by beparticular — APPROVED** [review]

**Pull request merged**